tests/integration/multinodes.nim

```diff
@@ -273,6 +273,13 @@ template multinodesuite*(name: string, body: untyped) =
         fail()
         quit(1)
 
+    proc updateBootstrapNodes(node: CodexProcess) =
+      without ninfo =? node.client.info():
+        # raise CatchableError instead of Defect (with .get or !) so we
+        # can gracefully shutdown and prevent zombies
```

Comment:
Graceful shutdown is for sure nice, but I think the OS would clean up the zombies, because the parent process (the test running process) will terminate anyhow. Just FYI.

Reply:
Yeah zombies aren't necessarily an issue. What won't happen with a `Defect` is execution of the except in `failAndTeardownOnError`, which runs teardown routines, fails the tests, and quits. The teardown routines stop the nodes (gracefully) and remove the data dirs for a clean start next run.